src/main/java/ladder/MainApplication.java

```diff
@@ -0,0 +1,37 @@
+package ladder;
+
+import ladder.domain.Ladder;
+import ladder.domain.generator.RandomLineGenerator;
+import ladder.domain.Users;
+import ladder.view.InputView;
+import ladder.view.OutputView;
+
+public class MainApplication {
+
+    public static void main(String[] args) {
+
+        Users users = inputUsers();
+        Ladder ladder = inputLadder(users);
+        OutputView.printResult(users, ladder);
```

Comment:
저는 위 로직을 봤을 때 아래처럼 읽혔어요 :)  
```
유저들을 받아 사다리를 입력받는다. (사다리 입력에 유저가 쓰이나?)
유저와 사다리의 결과를 출력한다. (사다리 행위 없이 생성시에 바로 결과가 나오는건가?) 
 ㄴ 코드 추적 20초 후.. (아 결과가 아니라 사다리를 그려주는거네?)
 ```
아래였으면 조금 더 자연스럽게 읽혔을 것 같아요. 
```suggestion
        Ladder ladder = createLadder(users);
        OutputView.printLadder(users, ladder);
```
이런 부분은 일종의 문학적인 감수성이고, 이건 사람마다 느끼는 게 다르기 때문에 참고만 하시라고 피드백 남깁니다! 하지만 **항상 독자 입장에서 신경써야 한다**는 건 좋은 코드의 핵심이기 때문에 요 마인드만 가져가셨으면 좋겠어요

Reply:
처음보는 사람도 한번에 이해할만한 코드! 명심해야 할 것 같습니다.